### ISSUE TITLE
L2CAP write request incomplete: 0 of 23

### ISSUE THREAD

**Issue by ericmigi** [nymea/nymea-networkmanager#39]
Very weird problem! Generally Berrylan is working great, except for one user.

Fresh install, fresh boot. Getting this error:
`L2CAP write request incomplete: 0 of 23`

iPhone 12 max
iOS 14.0.
Hardware: amlogic s912 tv box running Armbian on 5.9 kernel

also tested with an iPhone 8, on iOS 13.6.1. same issue. But it totally works for other users :/

![image](https://user-images.githubusercontent.com/1048265/97917358-38376e00-1d09-11eb-9709-0f7010e91bd8.png)

**Comment by t-mon** [comment]
Watching at your logs, the commands arrive successfully to the nymea-networkmanager, are you getting responses in the app, like network list or something? Are you able to set up a wifi network at all or what is the effect? 

Which Qt version do you have installed on your device? If this is an Armbian, I guess you have a buster underneath?

**Comment by ericmigi** [comment]
No, the network list never appears in the app. The spinner just keeps spinning. The weird part is that this exact setup works for other phones, it's only for this specific user that the error appears. He's tried with several different iPhones too!

Yes, it's stock Armbian buster.

**Comment by ericmigi** [comment]
this is happening on arm64, forgot to mention

**Comment by PastuDan** [comment]
I'm the user having this issue that @ericmigi is referring to. http://ix.io/2D8j are the logs when I run running nymea-networkmanager directly.

**Comment by mzanetti** [comment]
Can you please try to
a) Remove the RPi from the phone's bluetooth settings (if it's in there)
b) `sudo rm -r /var/lib/bluetooth/*` and reboot

FWIW, this is the line in QtBluetooth that bails out... Apparently when writing to the l2cap file provided by BlueZ.
https://code.qt.io/cgit/qt/qtconnectivity.git/tree/src/bluetooth/qlowenergycontroller_bluez.cpp#n979

No idea right now why this might happen for that particular device only... 

Does this phone work when trying to set up e.g. a Raspberry Pi with the original image from berrylan.org?

**Comment by PastuDan** [comment]
Sure!

a) I never paired it with my phone, so it's not in the list of devices
b) tried removing `/var/lib/bluetooth/*` on the pi... same issue still

---

After installing the official BerryLan image, I am still having the same issue :-( 

logs from official BerryLan image running on rPI4, using iPhone 11 Pro Max BerryLan app: http://ix.io/2Dgw
